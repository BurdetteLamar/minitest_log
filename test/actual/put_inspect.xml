<log>
  <summary verdicts='1' failures='0' errors='0'/>
  <data name='put_inspect' class='Range' method=':inspect'>
    0..3
  </data>
  <verdict method='verdict_assert_equal?' outcome='passed' id='__error_count__'>
    <expected class='Integer' value='0'/>
    <actual class='Integer' value='0'/>
  </verdict>
</log>
