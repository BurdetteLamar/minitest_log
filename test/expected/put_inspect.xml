<log>
  <summary verdicts='0' failures='0' errors='0'/>
  <data name='put_inspect' class='Range' method=':inspect'>
    0..3
  </data>
</log>
